armi/utils/tests/test_asciimaps.py

```diff
@@ -308,13 +364,40 @@ def test_hexFull(self):
             stream.seek(0)
             output = stream.read()
             self.assertEqual(output, HEX_FULL_MAP)
-        self.assertEqual(asciimap[0, 0], "0")
-        self.assertEqual(asciimap[0, -1], "2")
-        self.assertEqual(asciimap[0, -8], "6")
-        self.assertEqual(asciimap[0, 9], "4")
-        self.assertEqual(asciimap[-9, 0], "7")
-        self.assertEqual(asciimap[-8, 7], "8")
-        self.assertEqual(asciimap[6, -6], "3")
+
+        self.assertIn("7 1 1 1 1 1 1 1 1 0", str(asciimap))
```

Comment:
This doesn't necessarily have to change, but the same assertion is on line 347. Are we trying to show that `asciimap` hasn't been affected by the operations from 357 to 366, or is it unintentional?

Reply:
My comment on line 356 above:

"# also test writing from pure data (vs. reading) gives the exact same map"

I wanted to show there were no rotations or anything different between the two. Especially since we have bugs to that regard.